wof/examples/production_configs/wsgi.py

```diff
@@ -1,3 +1,4 @@
-#! /home/ubuntu/miniconda/envs/wofpy/bin/python
```

Comment:
Not your fault. I have my laptop setup to do the same. The only way to "defend" us against this is to add more tests. We need at least a full integration tests that tries to run an instance of `wofpy` in Travis-CI.